libr/debug/p/debug_native.c

```diff
@@ -1275,6 +1275,9 @@ static int r_debug_setup_ownership (int fd, RDebug *dbg) {
 static bool r_debug_gcore (RDebug *dbg, RBuffer *dest) {
 #if __APPLE__
 	return xnu_generate_corefile (dbg, dest);
+#elif __linux__ && __x86_64__
+	r_sandbox_disable (true);
```

Comment:
I need to disable it, otherwise I can't read from /proc. Or we change the code in libr/core/cmd_debug.c:

It's enabling sandbox before call the core-function:
```
        case 'g': // "dg"
                if (core->dbg->h && core->dbg->h->gcore) {
                        if (core->dbg->pid == -1) {
                                eprintf ("Not debugging, can't write core.\n");
                                break;
                        }
                        char *corefile = set_corefile_name (input + 1, core->dbg->pid);
                        eprintf ("Writing to file %s\n", corefile);
                        r_sandbox_creat (corefile, 0644);
                        RBuffer *file = r_buf_new_file (corefile);
                        if (!file) perror ("r_buf_new_file");
                        free (corefile);
                        r_sandbox_enable (true);
                        core->dbg->h->gcore (core->dbg, file);
                        r_buf_free (file);
                        r_sandbox_enable (false);
                }
                break;
```